VTEX - Marketplace APIs.json

```diff
@@ -2384,6 +2608,147 @@
+                            "nullable": true
+                        }
+                    }
+                }
+            },
+            "responseGetAffiliates":{
+                "type": "object",
+                "properties": {
+                    "id": {
+                        "description": "Affiliate ID.",
+                        "type": "string"
+                    },
+                    "followUpEmail": {
+                        "description": "Email of the store employee responsible for managing the order.",
+                        "type": "string"
+                    },
+                    "useSellerPaymentMethod": {
+                        "description": "If the store uses the seller payment method (`true`) or not (`false`).",
+                        "type": "boolean"
+                    },
+                    "name": {
+                        "description": "Affiliate name.",
+                        "type": "string"
+                    },
+                    "salesChannel": {
+                        "description": "Seller's [affiliate](https://help.vtex.com/en/tutorial/configuring-affiliates--tutorials_187) that will be associated with the marketplace's sales channel. Through the affiliate, the seller determines which of his sales channel will be used for the mapping.",
+                        "type": "string"
+                    },
+                    "searchURIEndpoint": {
+                        "description": "Search URI endpoint.",
+                        "type":  "string",
+                        "nullable": true
+                    },
+                    "searchURIEndpointVersion": {
+                        "description": "Search URI endpoint version.",
+                        "type": "string"
+                    },
+                    "searchURIEndpointAvailableVersions": {
+                        "description": "Search URI endpoint available versions.",
+                        "type": "array",
+                        "items": {
+                            "description": "Search URI endpoint available version.",
+                            "type": "string"
+                        }
+                    },
+                    "minimumValueAccumulated": {
+                        "description": "Minimum value accumulated in the cart to checkout the order.",
+                        "type": "integer",
+                        "nullable": true
+                    }
+                }
+            },
+            "requestUpdateAffiliate":{
+                "type": "object",
+                "properties": {
+                    "followUpEmail": {
+                        "description": "Email of the store employee responsible for managing the order.",
+                        "type": "string"
+                    },
+                    "useSellerPaymentMethod": {
+                        "description": "If the store uses the seller payment method (`true`) or not (`false`).",
```

Comment:
I am not sure but I don't think you can start a sentence with "If" if it's not a conditional sentence.